pycheribuild/projects/syzkaller.py

```diff
@@ -134,19 +149,34 @@ def install(self, **kwargs):
 
     def clean(self) -> ThreadJoiner:
         self.run_cmd(["chmod", "-R", "u+w", self.build_dir])
-        self.make_args.set_env(
-            HOSTARCH="amd64",
-            TARGETARCH=self.crosscompile_target.cpu_architecture.value,
-            TARGETOS="freebsd",
-            GOROOT=self.goroot.expanduser(),
-            GOPATH=self.gopath.expanduser(),
-            CC=self.CC, CXX=self.CXX,
-            PATH=self._new_path)
+        args = {
+            "HOSTARCH":"amd64",
+            "TARGETARCH":"arm64" if self.crosscompile_target.cpu_architecture.value == "aarch64"
+                else self.crosscompile_target.cpu_architecture.value,
+            "TARGETOS":"freebsd",
+            "CC":self.CC, "CXX":self.CXX,
+            "PATH":self._new_path
+        }
+        if self.use_go == GOROOTOptions.UPSTREAM:
+            args["GOROOT"] = self.goroot.expanduser()
+            args["GOPATH"] = self.gopath.expanduser()
+        self.make_args.set_env(**args)
 
         self.run_make("clean", parallel=False, cwd=self.gosrc)
         joiner = super().clean()
         return joiner
 
+class BuildMorelloSyzkaller(BuildSyzkaller):
+    target = "morello-syzkaller"
+    github_base_url = "https://github.com/CTSRD-CHERI/"
+    repository = GitRepository(github_base_url + "cheri-syzkaller.git", force_branch=True,
+        default_branch="morello-syzkaller")
+
+    def create_new_path(self, config):
+        # Replace sdk with morello-sdk:
+        self._new_path = (str(self.config.cheri_sdk_dir)[0:-3] + "morello-sdk/bin" + ":" +
```

Comment:
Use `self.config.morello_sdk_dir` instead. Also, `self.config.morello_sdk_dir` is Path so you can concatenate paths with `/`, e.g. `self.config.morello_sdk_dir / "bin"`.

Reply:
Also there should be a `morello_sdk_bindir` in the config object that points already to the binary directory